vnext/Microsoft.ReactNative/IReactDispatcher.h

```diff
@@ -6,9 +6,22 @@
 #include <dispatchQueue/dispatchQueue.h>
 #include <winrt/Microsoft.ReactNative.h>
 
+namespace Mso::React {
+
+struct __declspec(uuid("a8d9db25-3d16-4476-ae65-682fcee1260c")) ISimpleDispatch : ::IUnknown {
```

Comment:
>__declspec(uuid("a8d9db25-3d16-4476-ae65-682fcee1260c")) [](http://example.com/codeflow?start=7&length=56)

We can use the MSO_GUID instead.

Reply:
The reason is that the __declspec is not a standard C++ and using a macro we have an opportunity to use alternative implementations if required.